extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -90,6 +90,8 @@ require([
 	}
 
 	function configurePlayer(willAutoplay, willMute) {
+		win.guaSetCustomDimension(30, videoDetails.isDedicatedForArticle);
```

Comment:
how about setting the value of dimension to 'article' or 'wiki' to less confuse analytics?